src/Mod/Draft/draftguitools/gui_move.py

```diff
@@ -202,13 +202,37 @@ def move(self):
 
     def move_subelements(self):
         """Move the subelements."""
+        import Draft
```

Comment:
You don't need to include Draft. But if you need, do it at the top. All these Gui Commands will use Draft any way, so it's better to include it at the top of the module.